sql/core/src/test/resources/sql-tests/results/pipe-operators.sql.out

```diff
@@ -2442,6 +2411,238 @@ org.apache.spark.sql.catalyst.ExtendedAnalysisException
 }
 
 
+-- !query
+table windowTestData
+|> select cate, sum(val) over w
+   window w as (partition by cate order by val)
+-- !query schema
+struct<cate:string,sum(val) OVER (PARTITION BY cate ORDER BY val ASC NULLS FIRST RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW):bigint>
+-- !query output
+NULL	3
+NULL	NULL
+a	2
+a	2
+a	4
+a	NULL
+b	1
+b	3
+b	6
+
+
+-- !query
+table windowTestData
+|> select cate, sum(val) over w
+   window w as (order by val_timestamp range between unbounded preceding and current row)
+-- !query schema
+struct<cate:string,sum(val) OVER (ORDER BY val_timestamp ASC NULLS FIRST RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW):bigint>
+-- !query output
+NULL	5
+NULL	NULL
+a	13
+a	5
+a	5
+a	6
+b	13
+b	5
+b	8
+
+
+-- !query
+table windowTestData
+|> select cate, val
+    window w as (partition by cate order by val)
+-- !query schema
+struct<cate:string,val:int>
+-- !query output
+NULL	3
+NULL	NULL
+a	1
+a	1
+a	2
+a	NULL
+b	1
+b	2
+b	3
+
+
+-- !query
+table windowTestData
+|> select cate, val, sum(val) over w as sum_val
+   window w as (partition by cate)
+|> select cate, val, sum_val, first_value(cate) over w
+   window w as (partition by val)
+-- !query schema
+struct<cate:string,val:int,sum_val:bigint,first_value(cate) OVER (PARTITION BY val ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING):string>
+-- !query output
+NULL	3	3	NULL
+NULL	NULL	3	NULL
+a	1	4	a
+a	1	4	a
+a	2	4	a
+a	NULL	4	NULL
+b	1	6	a
+b	2	6	a
+b	3	6	NULL
+
+
+-- !query
+table windowTestData
+|> select cate, val, sum(val) over w1, first_value(cate) over w2
+   window w1 as (partition by cate), w2 as (partition by val)
```

Comment:
The `first_value` result is nondeterministic since the window `w2` has no ordering property. Please add an `order by` clause inside the window definition and regenerate the test result to make the test cases deterministic.